pyaerocom/aeroval/config/emep/reporting_base.py

```diff
@@ -515,79 +515,320 @@ def get_CFG(reportyear, year, model_dir) -> dict:
     ]
 
     # no new sites with 2021 observations (comment Svetlana T.)
+    # This list of stations was generated using a modified version of the script
```

Comment:
Please remove the outdated comment: `# no new sites with 2021 observations (comment Svetlana T.)`